Common/AllocBuffer.asm

```diff
@@ -5,7 +5,14 @@
 .include "Common/Common.s"
 .include "Online/Online.s"
 
+# Free old buffer if it exists
+  lwz r3,OFST_R13_SB_ADDR(r13)
```

Comment:
@JLaferri I was checking out the usage of this r13 offset and maybe it is being used too much by other files because _it is normally not being used for anything else_. 

However, if the code keeps using it like this, it might run eventually in the case where theres an implementation that relies of its value been persisted throught scenes, maybe? Making this fix a potentially dangerous, no? 

Maybe this warrants cleaning up the other places to try and find another r13 offset and use that one instead so this one is only used for this...

Reply:
Nope, I just meant potential future slippi uses only. 
A small concern of something that _could_ happen later.

Reply:
Gotcha. Yeah no I think the entire point of this offset is to provide a buffer that does not get overwritten across the scene.

It can be used for quick things where you don't need data to persist such as sending EXI commands, log output strings, etc. So I don't expect any problems going forward.